toml/src/test/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+
+class CratesLocalIndexServiceTest : BasePlatformTestCase() {
+    lateinit var cratesService: CratesLocalIndexService
+
+    override fun setUp() {
+        super.setUp()
+        cratesService = CratesLocalIndexServiceImpl.getTestInstance()
+    }
+
+    fun `test service is ready`() {
```

Comment:
Why do we need this test?
As I can see, it checks nothing because `CratesLocalIndexServiceImpl` is ready just after creation

Reply:
It ensures that the update of the index has been finished as the test gets to run. Test instance from `CratesLocalIndexServiceImpl.getTestInstance()` will update it on first access.

But I'm not sure if it is the right way to check this and if there is a need for this

Reply:
Note, currently `isReady` is not touched anywhere during `CratesLocalIndexServiceTest` except explicit call in `test service is ready` because of a way how test instance is initialized.